### ISSUE TITLE
Softpot midi signal cc + note on/off event

### ISSUE THREAD

**Issue by DJMathijs16** [tttapa/Control-Surface#228]
CCPotentiometer touchslider = {
  A14,                                   // Analog pin connected to potentiometer
  {0x08, CHANNEL_1}, // Channel volume of channel 1
};
I want to go from this:
[MIDILog.txt](https://github.com/tttapa/Control-Surface/files/4844753/MIDILog.txt)

To this:
[Traktor Touchstrip Log.txt](https://github.com/tttapa/Control-Surface/files/4844756/Traktor.Touchstrip.Log.txt)

So, instead of just a cc event, I want every time I touch my touchstrip:
1. When I press: A note on event
2. A cc event to determine the position of my finger
    (if not released yet: another cc event etc.)
3. When released: A note off event

So basically I want a combination of a button and a potentiometer.

**Comment by tttapa** [comment]
This is not supported by the CCPotentiometer class, you'll have to implement your own MIDI element. See [Custom-MIDI-Output-Element.ino](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html).
